modules/nlu/src/backend/typings.ts

```diff
@@ -38,7 +38,8 @@ export interface EntityExtractor {
 }
 
 export interface SlotExtractor {
-  train(trainingSet: Sequence[]): Promise<void>
+  load: Function
```

Comment:
forgot to put the right typings :p ?

Reply:
didn't forget .. found it weird to expose the loading internals but still wanted the autocomplete feature .. Will be fixed in the next refact PR